pkg/auth/login.go

```diff
@@ -0,0 +1,45 @@
+package auth
+
+import (
+	"os"
+
+	"github.com/openshift/origin/pkg/oc/cli/login"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+
+	"k8s.io/client-go/tools/clientcmd"
+	kclientapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+func Login(server, Username, Password, Token, CAAuth string, SkipTLS bool) error {
```

Comment:
Why params are capitalised?
can it be username, password, token, caAuth instead?